.github/workflows/ci.yml

```diff
@@ -269,6 +268,9 @@ jobs:
           name: masonry-snapshot-tests-${{ matrix.os }}
           path: masonry/src/widget/screenshots
 
+      - name: cargo test --doc
```

Comment:
Hmm. We likely do want these to produce screenshots - see #501.

Reply:
Well, it won't be a big task to move it if  it makes sense with the new screenshot implementation. As for right now, it looks like we don't even have an implementation, so the details may change.